src/models/v3/schema.ts

```diff
@@ -47,7 +47,7 @@ export class Schema extends BaseModel<v3.MultiFormatSchemaObject, { id?: string,
 
   $schema(): string {
     if (typeof this._schemaObject === 'boolean') return 'http://json-schema.org/draft-07/schema#';
-    return this._schemaObject.$schema || 'http://json-schema.org/draft-07/schema#';
+    return this._schemaObject.$schema ?? 'http://json-schema.org/draft-07/schema#';
```

Comment:
Is there an explicit case where you define the `$schema` field but with null or empty string? Just out of curiosity.

Reply:
No, at least not to my knowledge 🙂

Reply:
So then why changing this?

Reply:
Because sonar likes it better 🤷